cpu/esp32/Makefile.include

```diff
@@ -64,12 +66,18 @@ _ESP_WIFI_ANY_BASELIBS += -lphy -lstdc++
 # Baselibs needed when using esp_now module
 _ESP_NOW_BASELIBS = -lespnow -lmesh
 
-# NOTE: These checks can be turned into a normal conditional when #9913 is fixed
-BASELIBS += $(if $(filter esp_wifi_any,$(USEMODULE)),$(_ESP_WIFI_ANY_BASELIBS))
-BASELIBS += $(if $(filter esp_now,$(USEMODULE)),$(_ESP_NOW_BASELIBS))
+ifneq (,$(filter esp_wifi_any,$(USEMODULE)))
+  BASELIBS += $(_ESP_WIFI_ANY_BASELIBS)
+endif
 
-UNDEF += $(if $(filter cpp,$(FEATURES_USED)),$(BINDIR)/esp_cxx/cxa_guard.o)
-BASELIBS += $(if $(filter cpp,$(FEATURES_USED)),-lstdc++)
+ifneq (,$(filter esp_now,$(USEMODULE)))
+  BASELIBS += $(_ESP_NOW_BASELIBS)
```

Comment:
See the comment above. Unfortunatly, multiline suggestion don't work for deleted lines.
```suggestion
  BASELIBS += -lespnow -lmesh
```